src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -112,9 +112,9 @@ public override void Initialize([AllowNull] ViewWithData data)
                     // TODO: Do some decent logging here
                     return repositoryHost.LogOut();
                 })
-                .Catch<System.Reactive.Unit, Octokit.NotFoundException>(ex =>
+                .Catch<System.Reactive.Unit, Exception>(ex =>
                 {
-                    //this is caused when repository was deleted on github
+                    // Occurs on network error, when the repository was deleted on GitHub etc.
                     return Observable.Empty<System.Reactive.Unit>();
```

Comment:
Yeah, we should be... There was previously a TODO to add logging there. I might as well add it!